test/controllers/recipes_controller_test.rb

```diff
@@ -0,0 +1,44 @@
+require "test_helper"
+
+describe RecipesController do
+  # it "must be a real test" do
+  #   flunk "Need real tests"
+  # end
+
+  describe 'index' do
+    it "returns a list of search recipes results" do
+      query = "chicken"
+      VCR.use_cassette("recipes") do
```

Comment:
There are some interesting cases you're not covering here:

1. What happens if no search term is provided?
1. What if there are no paging parameters provided?
1. What if the paging parameters are invalid (negative number, etc)?
1. What if the paging parameters take you past the end of the available search results?